src/main/java/net/sf/jabref/logic/exporter/MetaDataSerializer.java

```diff
@@ -13,13 +20,42 @@
     /**
      * Writes all data in the format <key, serialized data>.
      */
-    public static Map<String, String> getSerializedStringMap(MetaData metaData) {
+    public static Map<String, String> getSerializedStringMap(MetaData metaData,
```

Comment:
How about refactoring this method a little bit? After all, it consists of various blocks, where different types of meta data contents are put into the final list. Each such block could be extract into a beautifully named private method :-)